ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -27,6 +27,7 @@ spec:
       default: $(context.pipelineRun.namespace)
       description: The namespace/organization all built images will be pushed to.
   workspaces:
+    - name: pipeline
```

Comment:
Why do we need this workspace here?

Reply:
This is added as per your earlier comment, this namespace is used in the ci pipeline and so, For now, we need to use this on a temporary basis.

Reply:
If we are using it on temporary basis, can we use some existing workspace?
CI pipeline uses the workspace named "pipeline" as place to clone repository and store results. Hosted pipeline has separate workspaces for that. I see no reason to add the new workspace to pipeline, especially that we would have to add it in a few places- README, webhook..

Reply:
Updated and moved under Repository workspace.